### PR TITLE
Prototype of student ambassador program registration page

### PR DESCRIPTION
**Pull request by polybuildr**

### CODE REVIEW

File: sql/create_table_sap_ambassadors.sql [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `sap_ambassadors`
+--
+
+CREATE TABLE IF NOT EXISTS `sap_ambassadors` (
+  `id` int(10) unsigned NOT NULL,
+  `name` varchar(64) NOT NULL,
+  `email` varchar(64) NOT NULL,
```
Comment (erilyth) [inline sql/create_table_sap_ambassadors.sql:32]: Would they also register with a password here itself?

File: sql/create_table_sap_ambassadors.sql [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `sap_ambassadors`
+--
+
+CREATE TABLE IF NOT EXISTS `sap_ambassadors` (
+  `id` int(10) unsigned NOT NULL,
+  `name` varchar(64) NOT NULL,
```
Comment (erilyth) [inline sql/create_table_sap_ambassadors.sql:31]: Phone number field :P

Comment (polybuildr) [inline sql/create_table_sap_ambassadors.sql:31]: Good catch. Will update PR.

File: sql/create_table_sap_ambassadors.sql (hunk 1 above)
Comment (polybuildr) [inline sql/create_table_sap_ambassadors.sql:32]: Hmm, good question. I asked Jeevan, and he told me that we'll take care of actual logging in once applications are reviewed and accepted.

**Comment by polybuildr** [comment]
Updated PR. @erilyth please recheck.

**Comment by erilyth** [comment]
Seems fine :+1:

**Comment by polybuildr** [comment]
Alright then, merging. :)

**Pull request merged**